src/textual/widgets/_text_area.py

```diff
@@ -755,15 +852,29 @@ def render_line(self, widget_y: int) -> Strip:
             A rendered line.
         """
         document = self.document
+        wrapped_document = self.wrapped_document
         scroll_x, scroll_y = self.scroll_offset
 
         # Account for how much the TextArea is scrolled.
-        line_index = widget_y + scroll_y
+        y_offset = widget_y + scroll_y
+
+        # If we're beyond the height of the document, render blank lines
+        out_of_bounds = y_offset >= wrapped_document.height
+        blank_line = Strip.blank(self.size.width)
```

Comment:
Should there be a style for the blank line (to set background color)?

Could we avoid constructing the blank line if we don't need it?

Reply:
We don't need to style the background colour because it's done at the TextAreaTheme level, which sets the background colour on `styles`.

I've removed the blank line variable and just create the strip on demand now.